### PR TITLE
Delete redundant util/fetch-params.sh

### PR DESCRIPTION
**Pull request by Liquid369**
We previously discussed fixing the `fetch-params.sh` script, but after discussion its not actually used and is redundant.
This script is broken and `install-params.sh` works fine, so lets just remove it.

### CODE REVIEW

File: src/init.cpp [hunk 1]
```diff
@@ -744,7 +744,7 @@ static void LoadSaplingParams()
         uiInterface.ThreadSafeMessageBox(strprintf(
                 _("Cannot find the Sapling parameters in the following directory:\n"
                   "%s\n"
-                  "Please run 'sapling-fetch-params' or './util/fetch-params.sh' and then restart."),
+                  "Please run 'sapling-fetch-params' or the included install-params.sh and then restart."),
```
Comment (Fuzzbawls) [inline src/init.cpp:747]: In order to cover both Windows zip based installs, as well as linux/macOS tarball installs, we should probably rewrite this entire section to be more modular. Here is a diff patch based on current `master` that splits up the string into multiple individual strings (with string replacement variables), then uses pre-compiler macros to craft the finalized UI message that the user will see.

```diff
Index: src/init.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/init.cpp b/src/init.cpp
--- a/src/init.cpp	(revision f07f32c09a99542f292154051fadd13c2cdee40f)
+++ b/src/init.cpp	(date 1682153190539)
@@ -741,11 +741,15 @@
     try {
         initZKSNARKS();
     } catch (std::runtime_error &e) {
-        uiInterface.ThreadSafeMessageBox(strprintf(
-                _("Cannot find the Sapling parameters in the following directory:\n"
-                  "%s\n"
-                  "Please run 'sapling-fetch-params' or './util/fetch-params.sh' and then restart."),
-                ZC_GetParamsDir()),
+        std::string strError = strprintf(_("Cannot find the Sapling parameters in the following directory:\n%s"), ZC_GetParamsDir());
+        std::string strErrorPosix = strprintf(_("Please run the included %s script and then restart."), "install-params.sh");
+        std::string strErrorWin = strprintf(_("Please copy the included params files to the %s directory."), ZC_GetParamsDir());
+        uiInterface.ThreadSafeMessageBox(strError + "\n"
+#ifndef WIN32
+                  + strErrorPosix,
+#else
+                  + strErrorWin,
+#endif
                                          "", CClientUIInterface::MSG_ERROR);
         StartShutdown();
         return;

```

**Review by Fuzzbawls — CHANGES_REQUESTED** [review]
Suggested a change that makes the error message more modular and which would cover windows users (who natively cannot run bash shell scripts).

Also, don't submit any kind of manual changes to `pivxstrings.cpp` or any of the various `.ts` files as they will only be overwritten during the scripted translation process.

**Review by Fuzzbawls — APPROVED** [review]
ACK 2c73d25f8fa080344ee34380ae8c965a45676e82

**Review by panleone — APPROVED** [review]
utACK 2c73d25f8fa080344ee34380ae8c965a45676e82

**Pull request merged**